inspirehep/modules/migrator/cli.py

```diff
@@ -42,7 +42,7 @@
 from inspire_utils.helpers import force_list
 
 from inspirehep.utils.schema import ensure_valid_schema
-from .models import InspireProdRecords
+from .models import LegacyRecordsMirror
```

Comment:
The name looks quite strange to me, for me a mirror is a process that does mirroring (that's 100% based on my experience of course). I'd name it something like `MigrationJob`, `MigrationTask`, `MigrationCachedRecord` or similar.
The migration is there because it's only used by that set of tasks/cli commands only.